app/views/candidate_interface/gcse/type/_form.html.erb

```diff
@@ -4,6 +4,10 @@
   <%= t("gcse_edit_type.page_titles.#{@subject}") %>
 </h1>
 
+<% if @subject == 'english' %>
+  <span class="govuk-hint govuk-!-font-size-19">This should be different from an English as a foreign language qualification.</span>
+<% end %>
```

Comment:
This hint should be associated with the legend for the radios (via `aria-describedby`), which is also the `h1` of the page, rather than having a separate "Type of qualification" legend. See [page in the prototype](https://apply-beta-prototype.herokuapp.com/application/12345/gcse/english?referrer=/application/12345/gcse/english/review)

I think you should be able to move it within the `govuk_radio_buttons_fieldset` method below, something like:

```ruby
<%= f.govuk_radio_buttons_fieldset :qualification_type, 
  legend: { text: t("gcse_edit_type.page_titles.#{@subject}"), size: 'l' }, 
  hint: { text: "This should be different from an English as a foreign language qualification." } 
  do %>
```

(then drop the h1 above)